query-engine/query-engine-node-api/Cargo.toml

```diff
@@ -20,13 +20,17 @@ driver-adapters = [
 anyhow = "1"
 async-trait.workspace = true
 query-core = { path = "../core", features = ["metrics"] }
-request-handlers = { path = "../request-handlers", features = ["native"] }
+request-handlers = { path = "../request-handlers", features = [
```

Comment:
Not really related to this target, but I think this was done by Serhii

Reply:
Note to @SevInf: Remember to create an issue about making `native`/`all` features consistent across crates. Sometimes `all` includes `native`, sometimes it doesn't.

Reply:
https://github.com/prisma/prisma-engines/issues/4785